params/variants.toml

```diff
@@ -1798,6 +1798,11 @@ rank = 2
 description = "Show ASCII grave (`` ` ``) as raised comma."
 selector.asciiGrave = 'raisedInverseComma'
 
+[prime.ascii-grave.variants.raised-turn-comma]
+rank = 3
+description = "Show ASCII grave (`` ` ``) as raised turned comma."
```

Comment:
May worth noting “identical to curly open single quote symbols (U+2018)”

Reply:
It's a useful hint, indeed, added!